### ISSUE TITLE
Component inline notification needs better design

### ISSUE THREAD

**Issue by LotharBlum** [infor-design/enterprise#1636]
@tmcconechy 
**Describe the bug**
The new inline notification needs better design and extended definition for theming

<img width="528" alt="grafik" src="https://user-images.githubusercontent.com/47633092/52788420-20b1e080-3061-11e9-92df-f613b31a8498.png">

**To Reproduce**
- background colors look dirty due to low opacity of color definition, which affects the clear and clean color of a message. You need to define very light step of related Soho colors independent of the background ...
- ... which brings me to the next point: there is no appearance of notifications for high contrast and dark UI defined. At least the ellipsis button is not reacting when I am trying to switch the theme.

**Expected behavior**
<img width="258" alt="grafik" src="https://user-images.githubusercontent.com/47633092/52789083-31fbec80-3063-11e9-86af-3c155b6f73a9.png">

**Screenshots**

**Platform**

**Additional context**

**Comment by tmcconechy** [comment]
@nickwynja what do you think we should do about the colors/design and highcontrast colors ? Rest we can handle

**Comment by clepore** [comment]
@nickwynja We should probably get a design review on this and since they are token colors, move this to design-system?

**Comment by tmcconechy** [comment]
Maybe split it, Make a design ticket for the design but there is a bug here that you cannot click the ... menu in the demo app as well.

**Comment by clepore** [comment]
@tmcconechy good call. I'd split the menu bug into a separate bug ticket as this is just an enhancement request.

**Comment by tmcconechy** [comment]
See https://github.com/infor-design/enterprise/issues/1639 for the bug part
